votingpool/db.go

```diff
@@ -56,13 +59,79 @@ type dbSeriesRow struct {
 	privKeysEncrypted [][]byte
 }
 
+// getUsedAddrBucketID returns the used addresses bucket ID for the given series
+// and branch. It has the form seriesID:branch.
+func getUsedAddrBucketID(seriesID uint32, branch Branch) []byte {
+	return bytes.Join(
```

Comment:
I did that here because I like to see all arguments in a single line, but when that's not possible I'll indeed use as much as possible of the first line before wrapping. I don't feel strong about it, though, so I'm happy to consistently use as much as possible of the first line if you think it's worth it